CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Puppet v3.12.2_7.6.2
+
+### Added
+
+- Update to Wazuh version 3.12.2_7.6.2
+
+## Wazuh Puppet v3.12.1_7.6.2
```

Comment:
3.12.1 won't be released due to some bugs related with RedHat feed, please remove it from the changelog.

Reply:
Done in 627c983